e2e/libs/backing_image/crd.py

```
@@ -0,0 +1,91 @@
+            all_running = True
+            backing_image_managers = self.list_backing_image_manager()            
+            for backing_image_manager in backing_image_managers["items"]:
+                current_state = backing_image_manager["status"]["currentState"]
+                name = backing_image_manager["metadata"]["name"]
+                logging(f"backing image manager {name} currently in {current_state} state")
+                if current_state != "running":
+                    all_running = False
+            if all_running is True:
+                return
+            time.sleep(self.retry_interval)
+        assert False, f"Waiting all backing image manager in running state timeout"
```

Ruff

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)